### DISCUSSION TITLE
"Can't import a named export from non ECMAscript module" and other config problems

### DISCUSSION THREAD

**Discussion by cmacloud01** [tailwindlabs/tailwindcss#5368] [Help]
Hey guys,

I am currently in the process of swapping from blueprint.js sidebar components to tailwindui components in a react app, and I am having a few problems:
1. When replacing the sidebar navigation component with the tailwind component, it did not appear properly styled in the browser.
2. After completely following all instructions for integrating tailwind into a react project, the app failed to compile in the browser, giving me this error:
"Can't import a named export from non ECMAscript module"

I'm hoping someone could possibly help walk me through what I'm doing wrong. 
I have screenshots of the relevant code if necessary.
We are using SASS, blueprint.js and react-app-rewired.

Thank you in advance!

**Comment by simonswiss** [comment]
Hey!

Sounds like this is possibly an issue with how `react-app-rewired` handles `.mjs` files. Do you happen to have any `.mjs` files in your project?

Maybe this will help?

https://stackoverflow.com/a/64907659

As per the sidebar not displaying properly, we'd probably need to see some code (not screenshots, but actual code reproduction of the problem in a GitHub repo) to be able to help properly 👍

**Reply by cmacloud01** [reply]
I'll check this out and get back to you! Thanks.
